config/env/tetris.yaml

```diff
@@ -11,6 +11,8 @@ height: 20
 pieces: ["I", "J", "L", "O", "S", "T", "Z"]
 # Allowed roations
 rotations: [0, 90, 180, 270]
+# Don't flatten if using CNN
+flatten: True
```

Comment:
If we move the flattening from the environment to the policy, then we don't need this, and we can simply remove it.